tool/tctl/common/auth_command.go

```diff
@@ -133,6 +137,9 @@ func (a *AuthCommand) Initialize(app *kingpin.Application, config *service.Confi
 	a.authRotate.Flag("manual", "Activate manual rotation , set rotation phases manually").BoolVar(&a.rotateManualMode)
 	a.authRotate.Flag("type", "Certificate authority to rotate, rotates host, user and database CA by default").StringVar(&a.rotateType)
 	a.authRotate.Flag("phase", fmt.Sprintf("Target rotation phase to set, used in manual rotation, one of: %v", strings.Join(types.RotatePhases, ", "))).StringVar(&a.rotateTargetPhase)
+
+	a.authLS = auth.Command("ls", "List connected auth servers")
+	a.authLS.Flag("format", "Output format: 'yaml', 'json' or 'text'").Default(teleport.YAML).EnumVar(&a.format)
```

Comment:
`EnumVar` needs some options, this is not going to work. Please give this a test on v10. I know zac suggested EnumVar but I think we're better off keeping this as a straight backport of https://github.com/gravitational/teleport/pull/14701, and creating a new PR to master changing this if we really want it